akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -1505,11 +1505,11 @@ private[stream] object Collect {
         super.onUpstreamFinish()
       }
 
-      override def onDownstreamFinish(): Unit = {
+      override def onDownstreamFinish(cause: Throwable): Unit = {
         if (isEnabled(logLevels.onFinish))
-          log.log(logLevels.onFinish, "[{}] Downstream finished.", name)
+          log.log(logLevels.onFinish, "[{}] Downstream finished.", name) // FIXME
```

Comment:
What needs fixing? Adding the cause to the log?

Reply:
Yes. Will do.